### ISSUE TITLE
Episodes not ordered by time

### ISSUE THREAD

**Issue by petrbela** [quidproquo/mr_video#6]
Why are the episodes [sorted by name](https://github.com/quidproquo/mr_video/blob/master/app/models/mr_video/cassette.rb#L24)? Wouldn't it make more sense to leave them in the order they were called? After all, it's just a sequence of remote requests. Or am I misunderstanding what the concept of episodes means?

**Comment by quidproquo** [comment]
@petrbela No real reason, but mainly due to the fact that a single rspec may test multiple endpoints of some API, hence a cassette may contain different episodes and they are easier to distinguish if they are sorted by endpoint url. In addition mr_video allows you to delete episodes so having episodes sorted by time may be hard to follow. Maybe the table should be sortable?

**Comment by petrbela** [comment]
Hmm, that's a good reason, although I feel that sorted by time is a much more common use case. You could certainly have a test case where you fetch data from multiple places but I think it still makes more sense to order them chronologically just by the simple fact that a cassette is a recording of events. Actually, if you change the order, VCR would fail because it expects the replay in a given sequence.

**Comment by quidproquo** [comment]
This is mainly a front-end concern and doesn't affect the underlying cassette, I'm closing this. If anyone wants to add a sorting mechanism to the table it would be appreciated.
